<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Directory Packager</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_folder_open">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>폴더 열기</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_path">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="group_work">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>331</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>311</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="radio_seq">
          <property name="text">
           <string>순차 작업</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_bal">
          <property name="text">
           <string>균등 작업</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_rnd">
          <property name="text">
           <string>랜덤 작업</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5,0,7">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>작업 개수</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_count"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>폴더명</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_folder"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_start">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>작업 시작</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="txt_status">
       <property name="text">
        <string>폴더 열기 버튼을 눌러서 폴더를 열어주세요.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>group_work</zorder>
   <zorder>txt_path</zorder>
   <zorder>btn_folder_open</zorder>
   <zorder>horizontalLayoutWidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_folder_open</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>folder_open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_start</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pack_start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>folder_open()</slot>
  <slot>pack_start()</slot>
 </slots>
</ui>
